index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portifolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="style">
+          <div class="right">
+           <h2>9:41</h2>
+          </div>
+          <div class="left">
+           <img src="/images/Cellular Connection.png" alt="">
+           <img src="/images/Wifi.png" alt="">
+           <img src="/images/Battery.png" alt="">
```

Comment:
- [ ] Please i would kindly suggest you to remove this i know it may confused you but not in the project template and not in project requirement its just a  screenshot